readme.md

```diff
@@ -23,3 +23,18 @@
 5) Run your tests
 
 To enable debugging, set `BUILDKITE_ANALYTICS_DEBUG_ENABLED=true`
+
+## Jest Collector Roadmap
```

Comment:
Given this repo is one day going to support other JS test frameworks, perhaps:

```suggestion
## Roadmap
````